include/constants/battle_config.h

```diff
@@ -226,8 +226,9 @@
 #define B_LAST_USED_BALL            TRUE     // If TRUE, the "last used ball" feature from Gen 7 will be implemented
 #define B_LAST_USED_BALL_BUTTON     R_BUTTON // If last used ball is implemented, this button (or button combo) will trigger throwing the last used ball.
 
-// Other
-#define B_DOUBLE_WILD_CHANCE        0     // % chance of encountering two Pokémon in a Wild Encounter.
+// Other settings
+#define B_DOUBLE_WILD_CHANCE                0    // % chance of encountering two Pokémon in a Wild Encounter.
+#define B_NEW_MULTI_BATTLE_DEFEAT_CONDITION TRUE // In Gen5+, multi battles end when the Player and also their Partner don't have any more Pokémon to fight.
```

Comment:
As I said in the discord, I recommend the name `B_MULTI_BATTLE_WHITEOUT`.

Reply:
Sorry I haven't logged in today. I had to re-reinstall Windows and I just didn't feel like logging back in until I had everything set up, WSL and the other stuff I use.

So, uh, I wouldn't be against that suggestion, but then what value should I assign to it?

Similar to Jaizu's suggestion, I feel I'd need to use an "OLD"/"NEW" approach, maybe? That would require defining an `OLD` and a `NEW` constants :eyes:

Reply:
> Sorry I haven't logged in today. I had to re-reinstall Windows and I just didn't feel like logging back in until I had everything set up, WSL and the other stuff I use.

It's fine man, no need to force yourself :)

> So, uh, I wouldn't be against that suggestion, but then what value should I assign to it?
> 
> Similar to Jaizu's suggestion, I feel I'd need to use an "OLD"/"NEW" approach, maybe? That would require defining an `OLD` and a `NEW` constants 👀

I forgot to clarify my suggestion: I would use `GEN_X` for this, seeing as it's a change introduced in Generation IV.